Task10_Auth/src/test/java/UpdateUserTests.java

```diff
@@ -0,0 +1,95 @@
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import reseponsesDTO.UpdateUserDto;
+import reseponsesDTO.UsersDto;
+
+import java.util.List;
+import java.util.Random;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class UpdateUserTests {
```

Comment:
Common comments:
1. Add one more test, when any required field is missed for "userToChange" field
2. In each test verify availability of zip codes
3. You are verifying that "userNewValues" user exists or not in the system, please update tests to verify the same for "userToChange" user

Reply:
1. Done
2. Added two validations, 1. old code is added or NOT added / 2, new code is removed or NOT removed
3. Added were applicable